src/Manifest/RepositoryRepositoryManifestReader.php

```diff
@@ -90,6 +87,42 @@ function (ModuleTransfer $moduleTransfer): string {
         return $manifests;
     }
 
+    /**
+     * @param \SprykerSdk\Integrator\Transfer\IntegratorCommandArgumentsTransfer $commandArgumentsTransfer
+     * @return array<string, null|string>
+     */
+    public function getFilterModules(IntegratorCommandArgumentsTransfer $commandArgumentsTransfer): array
+    {
+        $filterModules = [];
+
+        foreach ($commandArgumentsTransfer->getModules() as $module) {
+            $filterModules[sprintf('%s.%s', $module->getOrganization(), $module->getModule())] = $module->getVersion();
+        }
+
+        return $filterModules;
+    }
+
+    /**
+     * @param array<string, null|string> $filterModules
+     * @param string $moduleFullName
+     * @param string $fromVersion
+     * @param string $currentVersion
+     *
+     * @return bool
+     */
+    protected function shouldBeSkipped(array $filterModules, string $moduleFullName, string $fromVersion, string $currentVersion): bool
+    {
+        if (!$filterModules) {
+            return version_compare($fromVersion, $currentVersion, '>=');
+        }
```

Comment:
to cover it need to re-run all the manifests without specific modules in one test. It's overhead